core/common/src/main/scala/org/typelevel/otel4s/Attribute.scala

```diff
@@ -76,4 +76,8 @@ String, Boolean, Long, Double, List[String], List[Boolean], List[Long], List[Dou
   implicit def hashAttribute[T: Hash]: Hash[Attribute[T]] =
     Hash.by(a => (a.key, a.value))
 
+  implicit val hashAttributeExistential: Hash[Attribute[_]] = {
```

Comment:
Currently, the attribute value can exactly be one of: `Boolean`, `Double`, `String`, `Long`, `List[Boolean]`, `List[Double]`, `List[String]`, `List[Long]`.

So using a universal hashcode should be safe here.